.ci.sh

```diff
@@ -7,12 +7,14 @@ apt-get install -y apt-transport-https
 
 # Setup PCP
 wget -qO - https://bintray.com/user/downloadSubjectPublicKey?username=pcp | apt-key add -
```

Comment:
If we remove/comment the line below, this wget is no longer needed.  However,...